moveit_py:
  ros__parameters:
    planning_pipelines:
      pipeline_names: ["ompl"]   # string_array ✅
    plan_request_params:
      planning_pipeline: "ompl"

    ompl:
      planning_plugins: ["ompl_interface/OMPLPlanner"]
      request_adapters:          # 這裡一定要是陣列 ✅
        - default_planning_request_adapters/ResolveConstraintFrames
        - default_planning_request_adapters/ValidateWorkspaceBounds
        - default_planning_request_adapters/CheckStartStateBounds
        - default_planning_request_adapters/CheckStartStateCollision
      start_state_max_bounds_error: 0.1
